cachito/web/metrics.py

```diff
@@ -27,10 +27,23 @@ def init_metrics(app):
     )
     metrics.init_app(app)
     gauge_state = Gauge(
-        "cachito_requests_count", "Requests in each state", ["state"], multiprocess_mode="livesum"
+        "cachito_requests_count",
+        "Requests in each state",
+        ["state", "host"],
+        multiprocess_mode="livesum",
     )
     request_duration = Summary(
         "cachito_request_duration_seconds", "Time spent in in_progress state"
     )
     cachito_metrics["gauge_state"] = gauge_state
     cachito_metrics["request_duration"] = request_duration
+
+
+def requests_inc(state):
+    """Increase the number of requests in given state."""
+    cachito_metrics["gauge_state"].labels(state=state, host=hostname).inc()
+
+
+def requests_dec(state):
```

Comment:
Since we have type annotations in the checklist
```suggestion
def requests_inc(state: str) -> None:
    """Increase the number of requests in given state."""
    cachito_metrics["gauge_state"].labels(state=state, host=hostname).inc()


def requests_dec(state: str) -> None:
```